### ISSUE TITLE
Save button remains enabled after attachment is updated in a content

### ISSUE THREAD

**Issue by ashklianko** [enonic/lib-admin-ui#1988]
@sgauruseu commented on [Tue May 25 2021](https://github.com/enonic/app-contentstudio/issues/3189)

https://github.com/enonic/app-contentstudio/issues/1203

[pack.zip](https://github.com/enonic/app-contentstudio/files/6539557/pack.zip)
1. Upload renault.jpg
2. Open renault.jpg
3. Upload Dictyophorus_spumans01.jpg in the wizard

**BUG** - Save is enabled

**Comment by rymsha** [comment]
related anyhow https://github.com/enonic/xp/issues/6475 @alansemenov ?

**Comment by rymsha** [comment]
> related anyhow enonic/xp#6475

Yes. But server side trims default value itself, should be fixed in XP
